overrides/partials/footer.html

```diff
@@ -63,22 +63,17 @@
     <p>Developed by <a href="https://www.england.nhs.uk/">NHS England</a></p>
     <p>
       View the project on
-      <a href="https://github.com/NHSDigital/rap-community-of-practice"
-        >GitHub</a
-      >
+      <a href="https://github.com/NHSDigital/rap-community-of-practice">GitHub</a>
     </p>
+    <p><a href="{{ base_url }}/useful_links/">Useful Links</a></p>
```

Comment:
I think the useful links are a bit forgotten about down here!

Reply:
As this page contains mainly links to places people can learn more and get tips when doing RAP themselves, maybe a better home for it is in Implementing RAP.

I was trying to minimise the number of links on the main nav, I'm even thinking the Tags page could be moved into Site Info.

Reply:
I clicked on useful links from the footer and got a 404:
<img width="879" alt="image" src="https://github.com/user-attachments/assets/ca312de8-d2a3-4169-8d1d-d5a752160495">